### PR TITLE
Update Reader labels list

### PR DESCRIPTION
**Pull request by ajspotts**
Connects [#12979](https://github.com/department-of-veterans-affairs/caseflow/issues/12979)

### Description
Updates the list of reader labels to match the comprehensive list of document types stored in VBMS.
Prior to these changes, any document types which didn't have a corresponding label stored in DocumentTypes would simply return as "Other" leaving the user lacking necessary information.

### Acceptance Criteria

 - [ ] Documents returning from VBMS are returned with their correct assigned label

### CODE REVIEW

File: app/models/caseflow/document_types.rb [hunk 1]
```diff
@@ -2,48 +2,364 @@
 module Caseflow
   module DocumentTypes
     TYPES = {
-      10  => "UNKNOWN",
+      10 => "UNKNOWN",
```
Comment (hschallhorn) [inline app/models/caseflow/document_types.rb:5]: Can we sort these numerically? (11 would follow 10)

Comment (ajspotts) [reply app/models/caseflow/document_types.rb:5]: Certainly I was just following the pattern that had been previously set but happy to re-sort!

File: app/models/caseflow/document_types.rb [hunk 2]
```diff
@@ -74,8 +390,7 @@ module DocumentTypes
       167 => "VA 26-1880 Request for Certificate of Eligibility for VA Home Loan Benefits",
       168 => "VA 26-4555 Veterans Application in Acquiring Specially Adapted Housing or Special Home Adaptation Grant Under Title 38 U.S.C. 2101(a) or (b)",
       169 => "VA 21-3537a Field Examination Request",
-      17  => "VAMC Report of Hospitalization",
-      171 => "VA 28-1900 Disabled Veterans Application for Vocational Rehabilitation",
```
Comment (hschallhorn) [inline app/models/caseflow/document_types.rb:78]: I'm wondering whether or not we should be removing these. While the forms may not exist anymore, they certainly did at one point and probably exist in the documents of older appeals. 
@araposo-tistatech Thoughts on whether or not we should keep recognizing labels that have a strikethrough in the spreadsheet?

File: app/models/caseflow/document_types.rb [hunk 3]
```diff
@@ -144,23 +188,17 @@ module DocumentTypes
...
       377 => "VA 21-4140 Employment Questionnaire",
       378 => "VA 22-1990 Application for Education Benefits",
-      38  => "DD 1515 Pay and Allotment Information--Deceased Member",
       380 => "MAPD Contention Screen",
       381 => "VA 21-0781, Statement in Support of Claim for PTSD",
-      382 => "VA 21-0781a, Statement in Support of Claim for PTSD Secondary to Sexual Assault",
+      382 => "VA 21-0781a, Statement in Support of Claim for PTSD Secondary to Sexual Personal Assault",
       383 => "VA 21-8767 Death Pension Award Attachment",
       384 => "VA 21-8768 Disability Pension Award attachment",
-      385 => "VA 21-8951-2 Notice of Waiver of VA Comp or Pension to Receive Military Pay and Allowances",
```
Comment (hschallhorn) [inline app/models/caseflow/document_types.rb:160]: Based on [this](https://github.com/department-of-veterans-affairs/caseflow/issues/12979#issuecomment-607907473) comment from Ann-Marie, we should be leaving docs that don't exist in the spreadsheet.

File: app/models/caseflow/document_types.rb [hunk 4]
```diff
@@ -478,36 +578,471 @@ module DocumentTypes
...
+      943 => "VETSNET Award Print",
+      944 => "DBQ - Photographs",
+      945 => "DBQ - Imaging Reports",
+      946 => "DBQ - Other Test Results",
+      947 => "Photographs - Veteran Provided",
+      948 => "Imaging Reports - Veteran Provided",
+      949 => "Other Test Results - Veteran Provided",
       950 => "RAMP Opt-In Notice",
       951 => "RAMP Opt-in Election",
-      999_981 => "C&P Exam XML"
```
Comment (hschallhorn) [inline app/models/caseflow/document_types.rb:509]: Based on [this](https://github.com/department-of-veterans-affairs/caseflow/issues/12979#issuecomment-607907473) comment from Ann-Marie, we should be leaving docs that don't exist in the spreadsheet.

File: app/models/caseflow/document_types.rb (hunk 2 above)
Comment (araposo-tistatech) [reply app/models/caseflow/document_types.rb:78]: @hschallhorn yes, I agree (regrets if my original response was not clear enough). I believe these documents will probably still exist in some files (most likely legacy) and we should still be able to recognize the labels.

File: app/models/caseflow/document_types.rb (hunk 3 above)
Comment (ajspotts) [reply app/models/caseflow/document_types.rb:160]: re-added!

File: app/models/caseflow/document_types.rb (hunk 4 above)
Comment (ajspotts) [reply app/models/caseflow/document_types.rb:509]: this one as well!

File: app/models/caseflow/document_types.rb (hunk 2 above)
Comment (ajspotts) [reply app/models/caseflow/document_types.rb:78]: re-added all labels that were struckthrough on the master list!

File: app/models/caseflow/document_types.rb [hunk 5]
```diff
@@ -74,17 +136,15 @@ module DocumentTypes
       167 => "VA 26-1880 Request for Certificate of Eligibility for VA Home Loan Benefits",
       168 => "VA 26-4555 Veterans Application in Acquiring Specially Adapted Housing or Special Home Adaptation Grant Under Title 38 U.S.C. 2101(a) or (b)",
       169 => "VA 21-3537a Field Examination Request",
-      17  => "VAMC Report of Hospitalization",
-      171 => "VA 28-1900 Disabled Veterans Application for Vocational Rehabilitation",
       172 => "VA 551 Application for Accrued Benefits by Veterans Spouse, Child or Dependent Parent",
+      171 => "VA 28-1900 Disabled Veterans Application for Vocational Rehabilitation",
```
Comment (hschallhorn) [inline app/models/caseflow/document_types.rb:140]: 171 and 172 out of order

File: app/models/caseflow/document_types.rb [hunk 6]
```diff
@@ -381,27 +405,24 @@ module DocumentTypes
...
-      61  => "Marriage Certificate / License",
       610 => "Competency Review Request Memorandum",
       611 => "Congressional Correspondence Fiduciary",
       612 => "Court Waiver or Consent PGF",
       614 => "Court/Legal Documents PGF",
       615 => "Direct Deposit Enrollment Request Fiduciary - Voided Check",
       616 => "Expense Request - Denial",
       617 => "Fiduciary Appeal - Additional Information Required Letter",
-      618 => "Fiduciary Appeal - Case forwarded to Board of Veterans Appeals",
+      618 => "Fiduciary Appeal - Case  forwarded to Board of Veterans Appeals",
```
Comment (hschallhorn) [inline app/models/caseflow/document_types.rb:415]: Extra space

File: app/models/caseflow/document_types.rb [hunk 7]
```diff
@@ -478,36 +584,504 @@ module DocumentTypes
...
+      901 => "VR-27 Six-Credit Hour Exclusion Notification",
+      902 => "VR-28 Course Reduction or Withdrawal",
+      903 => "VR-29 Apportionment Letter - Original - Guardian",
+      904 => "VR-30 Apportionment Letter - Reinstatement - Guardian",
+      905 => "VR-31 Approved Tutor Contract Notification",
+      906 => "VR-32 IEAP Development Prior Completion of Training",
+      907 => "VR-33 Referral - Employment Services",
+      908 => "VR-35 Discontinuance Letter - Plan Developed",
+      909 => "VR-36 Discontinuance Letter - Failure to Cooperate-Unsatisfactory Conduct",
+      910 => "VR-39 Rehabilitation  Letter - Employment",
```
Comment (hschallhorn) [inline app/models/caseflow/document_types.rb:666]: Extra space

File: app/models/caseflow/document_types.rb [hunk 8]
```diff
@@ -478,36 +584,504 @@ module DocumentTypes
...
+      1169 => "VA Form 28-1904 Agreement to Train OJT Disabled Veterans",
+      1170 => "VA Form 28-1917 Monthly Statement of Wages paid to Trainee",
+      1171 => "VA Form 28-8739a Protection of Privacy",
+      1172 => "VA Form 28-8832 Application for Counseling",
+      1173 => "VA Form 2237 Request, Turn in and Receipt for Property of Services-Purchase Request",
+      1174 => "VA Form 5655 Financial Status Report",
+      1175 => "VA Form 21-0304 Application for Benefits for Certain Children with Disabilities Born of Vietnam and Certain Korea Service Veterans",
+      1176 => "VA Form 22-1990 Application for VA Education Benefits",
+      1177 => "VA Form 22-1990e Application for Family Member to use Transferred Benefits",
+      1179 => "VA Form 22-5490 Dependents Application for VA Education Benefits",
```
Comment (hschallhorn) [inline app/models/caseflow/document_types.rb:932]: Missing 
```ruby
      1178 => "VA Form 22-1999b Notice of Change in Student Status",
```

Comment (ajspotts) [reply app/models/caseflow/document_types.rb:933]: apologies for the oversights, thanks for catching them!

**Pull request merged**